website/docs/r/cdn_frontdoor_firewall_policy.html.markdown

```diff
@@ -260,6 +264,16 @@ A `managed_rule` block supports the following:
 
 ---
 
+A `log_scrubbing` block supports the following:
+
+* `enabled` - (Optional) Is log scrubbing enabled? Possible values are `true` or `false`. Defaults to `true`.
+
+* `rule` - (Optional) One or more `scrubbing_rule` blocks as defined below.
```

Comment:
```suggestion
* `rule` - (Optional) One or more `rule` blocks as defined below.
```

Reply:
Same as above, the `managed_rule` also uses `rule` so I gave it a different name in the documentation trying to not confuse the end user when they are looking up what the `scrubbing_rule` block contains.